microshift_storage/container_storage_interface_microshift/microshift-persistent-storage-csi.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="persistent-storage-csi-microshift"]
+= Configuring CSI volumes for MicroShift
+include::_attributes/attributes-microshift.adoc[]
+:context: persistent-storage-csi-storage-microshift
+
+toc::[]
+
+The Container Storage Interface (CSI) allows {product-title} to consume
```

Comment:
I would normally call out the hard wrapping here, but I see that you pulled this from the existing storage's assembly. I'll assume this is mostly the case throughout the rest of the PR and won't point out anymore. Fix if you want, but won't insist on it :)